app/styles/components/list.css

```
@@ -54,8 +54,8 @@
 }
 
 .list--disc li::before {
-    --list-marker-bg-color: currentColor;
-  background: conic-gradient(currentColor, var(--surface));
+    --list-marker-bg-color: currentcolor;
+  background: conic-gradient(currentcolor, var(--surface));
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)